content/docs/0.14.x/reference/files/sli/index.md

```diff
@@ -1,36 +1,47 @@
 ---
-title: Service-Level Indicators (SLI)
+title: sli.yaml
```

Comment:
Not sure about this renaming, but no strong objections

Reply:
I am still learning the Keptn jargon so welcome feedback.  I tend to go for the filename because it is something I can find on my system but that's just me...

Reply:
Let's keep it as is and then see how to improve overall